.github/workflows/ci_dockers.yml

```diff
@@ -135,15 +135,15 @@ jobs:
           push: false
         timeout-minutes: 50
 
-  build-nvidia:
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Build NVIDIA Docker
-        uses: docker/build-push-action@v2
-        with:
-          file: dockers/nvidia/Dockerfile
-          push: false
-        timeout-minutes: 50
+#  build-nvidia:
```

Comment:
would probably be a good idea to also add an extra comment stating why this is disabled